src/components/Shared/CalendarPicker.vue

```diff
@@ -55,6 +55,12 @@ export default {
 			// for calendars where only one calendar can be selected, disable if there are < 2
 			return this.multiple ? this.calendars.length < 1 : this.calendars.length < 2
 		},
+		options() {
+			return this.calendars.map(calendar => {
```

Comment:
You could just reuse your new method in the map, right ?

Reply:
True 😂